File: migrations/NASDAQ.py
import os
import pandas
from init_db import get_db

def main():
    db = get_db()
    nasdaq = db.NASDAQ
    nasdaq.drop()
    csv = pandas.read_csv('migrations/companylist.csv')
    csv =  csv.drop(['LastSale','ADR TSO', 'IPOyear','Unnamed: 9' ], 1)
    csv.columns = ['symbol', 'name', 'market_cap', 'sector', 'industry', 'summary_quote']
    documents = csv.to_dict('index')
    documents = documents.values()
    nasdaq.insert_many(documents)

main()

File: models/NASDAQ.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var nasdaqSchema = new Schema( {
          sector: String,
          symbol: String,
          summary_quote: String,
          market_cap: String,
          name: String,
          industry: String
})

module.exports = mongoose.model('NASDAQ', nasdaqSchema, 'NASDAQ');
